### ISSUE TITLE
Invalid face attribute :foreground nil

### ISSUE THREAD

**Issue by oatmealm** [jkitchin/ov-highlighter#13]
Hi there! I've started seeing these messages, possibly after switching back to helm from ivy, when trying to apply forgoround color. Background and other markup works as expected otherwise.

```
Invalid face attribute :foreground nil
```

![gnome-shell-screenshot-OPVZ10](https://user-images.githubusercontent.com/68159077/114854044-b3d33600-9de4-11eb-8b87-286fed865274.png)

**Comment by jkitchin** [comment]
I can't reproduce this, how did you switch from ivy to helm?

**Comment by oatmealm** [comment]
I’m using Doom so it’s a matter of choosing between either options. Do you need me to look for something specific? It seems that some highlighting functions work (red fg) while others don’t (yellow fg). 

Thanks.

**Comment by jkitchin** [comment]
I don't have a yellow fg ov highlight function.

I guess this is not working for helm. It doesn't seem to work correctly with `completing-read`. Off the top of my head, I am not sure why.  It does work with ivy for me. At least this is the case for the foreground function, and maybe any function that uses completing-read, which includes ov-highlight-color, foreground and font.

you should be able to make any color you want like

    (ov-highlight-make "red-fg" '(:foreground "red"))

which creates `ov-highlight-red-fg` that you can call.

**Comment by oatmealm** [comment]
Yes I've been using the custom functions for this purpose. I've noticed in another module  that facemenu.el is not loading for some reason, which was not the case in doom so far. I'm loading it manually to test. Could this be related ie ov-highlight loading before facemenu (if there's a dependency there) ?

**Comment by jkitchin** [comment]
I am unaware of a facemenu dependency
